### ISSUE TITLE
Page breaks are getting ignored

### ISSUE THREAD

**Issue by sonovice** [rism-ch/verovio#728]
I am trying to render a 2 page mei file using the following (Python) code:

```python
import verovio

vrv = verovio.toolkit()
vrv.loadFile('breaks.mei')
vrv.renderToSvgFile('output_1.svg', 1)
```

For some reason the `<pb>` element (line 1500) is ignored and verovio tries to put everything on a single page. Executing `print(vrv.getPageCount())` results in the output `1` instead of `2`. Maybe I'm doing something wrong?

[breaks.mei (zipped)](https://github.com/rism-ch/verovio/files/1526626/breaks.zip)

**Comment by lpugin** [comment]
The `<section>` content needs to start with a `<pb/>`, and then encode system breaks with `<sb/>`

**Comment by sonovice** [comment]
Thanks for the quick answer? Is there a reason for the required page break at the start of the section?

**Comment by lpugin** [comment]
As far as I know this follows the TEI practice.

**Comment by sonovice** [comment]
Alright, thanks!

**Comment by pe-ro** [comment]
Technically, `<pb>` signals the beginning of a page, not a break between pages.  Therefore, the file content should begin with a `<pb>`.  But, if there isn't one, it can/should be assumed.

**Comment by sonovice** [comment]
Hm, as far as I know, Verovio switches in a different "break mode" as soon as a `<pb>` or `<sb>` element occours. So this might not be the best solution.

**Comment by pe-ro** [comment]
Let me rephrase -- I meant the initial `<pb>` is required when applicable; that is, where you want to enforce a certain layout.  When other `<pb>` elements are used, there should also be one before the content.  If there isn't, one can be assumed.

Better?

**Comment by lpugin** [comment]
We could. But I think this would be reinforcing the confusion around the idea that `<pb/>` encode the beginning of a page (and not a page break). So I would rather not change it and encourage users to adjust their encoding accordingly.
